packages/util/json/index.ts

```diff
@@ -10,5 +10,20 @@ export const replacer = {
       return number.toHex(v)
     }
     return v
+  },
+  uint8ArrayToHexString: (_: string, v: any) => {
+    // Filtering out properties
+    if (v instanceof Uint8Array) {
+      return Buffer.from(v).toString("hex")
+    }
+    return v
   }
 }
+
+export function format(
+  data: any,
+  replacer: (this: any, key: string, value: any) => any | null = null
+): string {
+  const json = JSONB({ useNativeBigInt: true })
+  return json.stringify(data, replacer, 2)
```

Comment:
Maybe we can do one step further here: (the `replacer` argument may also be renamed to avoid conflict with exported `replacer` object)

```suggestion
  return json.stringify(
      data, 
      (key: string, value: any) => {
          return replacer.uint8ArrayToHexString(key, replace(key, value))
      }, 
      2
  )
```